server/src/main/resources/server-reference.conf

```diff
@@ -53,6 +53,10 @@ server.authMode = "none" # none, basic_auth or simple_login
 server.authUsername = ""
 server.authPassword = ""
 
+# Sessions
+# 0 = default session cookie behavior (no persistent Max-Age); >0 = persistent JSESSIONID with given max-age in minutes
+server.sessionCookieMaxAgeMinutes = 0
```

Comment:
I would say put it under authentication and name it `simpleAuthSessionCookieMaxAge`, add a comment after it similar to the other comments